### ISSUE TITLE
lack of some parameters' introduction

### ISSUE THREAD

**Issue by llplmlyd** [pingcap/docs-cn#4970]
File: [/daily-check.md](https://docs.pingcap.com/zh/tidb/stable/daily-check)
i want to check some parameter if affects, but the 4.0 version doc  doesn't match any. like these:**learner-peer-region-count**
![image](https://user-images.githubusercontent.com/33628991/100412995-21d6c500-30b1-11eb-8fa7-eafa4f983b69.png)

and in the Operator panel  of PD there i also need some introductions to judge my cluster if it is in the safety status.
![image](https://user-images.githubusercontent.com/33628991/100413156-9f9ad080-30b1-11eb-870c-92137fa49187.png)

**Comment by ran-huang** [comment]
Thank you for your feedback!
- For `learner-peer-region-count`, you may refer to [this answer](https://asktug.com/t/topic/2407/2).
- For the **Operator** panel, check out [this doc](https://docs.pingcap.com/zh/tidb/stable/grafana-pd-dashboard#operator). 😸

**Comment by llplmlyd** [comment]
right. first, wana some more detail about the reason of rise or fail, and the second problem is ,i had seen this doc, but there does not have this parameter
![image](https://user-images.githubusercontent.com/33628991/100416495-8f86ef00-30b9-11eb-9c71-2496b45fd7ea.png)

**Comment by ran-huang** [comment]
@kissmydb PTAL~ Thanks!
Do you think we need to add more detailed info in our docs? It seems that many monitoring items are missing 🤔

**Comment by llplmlyd** [comment]
yes, if can show more details, do can locate problem more quickly and conveniently.
